packages/bsky/src/services/util/search.ts

```diff
@@ -22,7 +22,7 @@ export const getUserSearchQuery = (
     limit,
     cursor,
     direction: 'asc',
-    keyset: new SearchKeyset(distanceAccount, ref('handle')),
+    keyset: new SearchKeyset(distanceAccount, ref('actor.did')),
```

Comment:
Not necessarily a bad thing, but this will affect the feel of search results in some cases, since it was sometimes pretty clear they were alphabetical by handle.  It's not uncommon for the distance to be `0` for many matches, especially with short/common `term`s.

In any case, I don't think we need to address it, just a note!

Reply:
I had to adjust this to make the type system happy, but I'd be happy to adjust another way if you think that'd be happy

I feel like that's okay tho 🤔 we need to fix up search in general so that it takes into account other ranking factors - proximity, activity level, etc. so results won't always be sorted alphabetical by handle